### PR TITLE
Multiple Tickets

### PR DESCRIPTION
**Pull request by schlenkibus**

### CODE REVIEW

File: projects/NonMaps/src/main/java/com/nonlinearlabs/client/world/maps/parameters/PlayControls/MacroControls/Macros.java [hunk 1]
```diff
@@ -70,6 +70,24 @@ public ModSource toModSource() {
 			}
 			return ModSource.None;
 		}
+
+		public static String fromParameterID(int mcNum) {
```
Comment (hhoegelo) [inline projects/NonMaps/src/main/java/com/nonlinearlabs/client/world/maps/parameters/PlayControls/MacroControls/Macros.java]: Consider ModulateableParameterModel.ModSource.fromParameterId().toString()

File: projects/NonMaps/src/main/java/com/nonlinearlabs/client/dataModel/editBuffer/MacroControlParameterModel.java [hunk 2]
```diff
@@ -6,7 +6,6 @@
 
 public class MacroControlParameterModel extends BasicParameterModel {
 	public StringDataModelEntity givenName = new StringDataModelEntity();
-	public StringDataModelEntity info = new StringDataModelEntity();
```
Comment (hhoegelo) [inline projects/NonMaps/src/main/java/com/nonlinearlabs/client/dataModel/editBuffer/MacroControlParameterModel.java]: Don't we need info texts for MacroControls anymore?

File: projects/NonMaps/src/main/java/com/nonlinearlabs/client/world/overlay/ParameterInfoDialog.java [hunk 3]
```diff
@@ -206,43 +207,24 @@ protected void update(ParameterPresenter selectedParameter) {
...
-		// while (parent != null) {
-		// if (parent instanceof ParameterGroupIface) {
-		// ParameterGroupIface group = (ParameterGroupIface) parent;
-		// String longText = group.getName().getLongName() + " \u2013 " +
-		// p.getName().getLongName();
-		// return longText;
-		// }
-		// parent = parent.getParent();
-		// }
-		// }
```
Comment (hhoegelo) [inline projects/NonMaps/src/main/java/com/nonlinearlabs/client/world/overlay/ParameterInfoDialog.java]: Is the empty string intended here?

File: projects/playground/src/groups/MacroControlsGroup.cpp [hunk 4]
```diff
@@ -122,3 +122,9 @@ bool MacroControlsGroup::isMacroTime(const ParameterId &id)
 {
   return (id.getNumber() >= 324 && id.getNumber() <= 327) || (id.getNumber() == 370 || id.getNumber() == 372);
 }
+
+bool MacroControlsGroup::isMacroControl(const int paramNumber)
```
Comment (hhoegelo) [inline projects/playground/src/groups/MacroControlsGroup.cpp]: consider introducing 
static com.nonlinearlabs.client.dataModel.editBuffer.ParameterFactory.isMacroControl(), using the already existing macroControls array.

File: projects/playground/src/proxies/hwui/panel-unit/boled/preset-screens/controls/PresetListHeaderArrow.cpp [hunk 5]
```diff
@@ -50,5 +50,5 @@ Font::Justification PresetListHeaderArrow::getJustification() const
 
 std::shared_ptr<Font> PresetListHeaderArrow::getFont() const
 {
-  return Oleds::get().getFont("Emphase_9_Symbol", getFontHeight());
+  return Oleds::get().getFont("Emphase-9-Regular", getFontHeight());
```
Comment (hhoegelo) [inline projects/playground/src/proxies/hwui/panel-unit/boled/preset-screens/controls/PresetListHeaderArrow.cpp]: Symbol -> Regular intended?

Comment (schlenkibus) [reply projects/playground/src/proxies/hwui/panel-unit/boled/preset-screens/controls/PresetListHeaderArrow.cpp]: Yes, the Symbol font is currently deprecated and can hopefully be removed from Thursday when the Emphase-X-Regular font gets updated with currently missing Up/Down Arrows. Arrow left/right is correct in choosing Regular font.

File: projects/NonMaps/src/main/java/com/nonlinearlabs/client/dataModel/editBuffer/MacroControlParameterModel.java (hunk 2 above)
Comment (schlenkibus) [reply projects/NonMaps/src/main/java/com/nonlinearlabs/client/dataModel/editBuffer/MacroControlParameterModel.java]: MC Info texts start empty and can be overwritten by the user -> this data is also being passed along as 'normal' parameter info, the removed field was present in BasicParameterModel and MacroControlParameterModel.

File: projects/NonMaps/src/main/java/com/nonlinearlabs/client/world/overlay/ParameterInfoDialog.java (hunk 3 above)
Comment (schlenkibus) [reply projects/NonMaps/src/main/java/com/nonlinearlabs/client/world/overlay/ParameterInfoDialog.java]: fixed

**Pull request merged**